spec/util/java_version_spec.rb

```diff
@@ -27,6 +27,10 @@
     expect(mod.bad_java_version?("1.8.0-beta")).to be_falsey
   end
 
+  it "should not mark non-standard javas as bad (IBM JDK)" do
+    expect(mod.bad_java_version?("pwi3270sr9fp10-20150708_01 (SR9 FP10)")).to be_falsey
```

Comment:
Yes, that was pulled from this bug report https://github.com/elastic/logstash/issues/3677#issuecomment-136339776